packages/vue/src/components/range/RangeSlider.jsx

```diff
@@ -131,14 +130,23 @@ const RangeSlider = {
     this.setQueryListener(this.$props.componentId, onQueryChange, null);
   },
   beforeMount(){
+
     this.addComponent(this.$props.componentId);
+    this.addComponent(this.internalComponent);
```

Comment:
What is `this.internalComponent` doing? It is not initialized

Reply:
Sure,then you can git rid of it.